projects/burnr/src/components/AccountMenu.tsx

```diff
@@ -42,68 +51,59 @@ const AccountMenu: React.FunctionComponent = () => {
 	};
 
 	const { userAddress, userJson, userSeed } = account;
+
+	const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+	const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
 	
 	return (
-		<Grid
-			container
-			direction='column'
-		>
-			<Grid item className={classes.section}>
-				<Grid item xs={12}>
+		<>
+			<IconButton onClick={handleClick}>
+				<ExpandMoreIcon style={{color: grey[500]}}/>
+			</IconButton>
+
+			<Menu
+				transformOrigin={{vertical: -40, horizontal: 'left'}}
+				anchorEl={anchorEl}
+				keepMounted
+				open={Boolean(anchorEl)}
+				onClose={handleClose}
```

Comment:
Instead of calling a function that will call the setter its better to do:
```
onClose={() => setAnchorEl(null)}
```
and remove function `handleClose` from above